.github/workflows/go.yml

```diff
@@ -15,11 +15,37 @@ on:
       - '*.md'
 
 jobs:
+  cover:
+    name: Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          cache: true
+          go-version: '1.22'
+
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Run go test
+        run: go test -coverprofile=coverage.txt -covermode=atomic ./...
```

Comment:
and no `race` here?

Reply:
This one is about coverage.

Reply:
so 2 runs with `race` and one run with no `race`?